components/base/Collapsible.tsx

```diff
@@ -78,15 +85,33 @@ function Collapsible({ triggerText, children }: Props) {
 
   const transitionTime = 200;
 
+  // Sets active state on collapsible
+  function handleActive() {
+    if (!active) {
+      setActive(true);
+    } else {
+      setActive(false);
+    }
+  }
+
+  // Handle "Enter" key to open/close collapsible
+  function handleKeyDown(e) {
+    if (e.keyCode === 13) handleActive();
+  }
+
   return (
-    <CollapsibleWrapper>
+    <CollapsibleWrapper
+      tabIndex={0}
+      onKeyDown={(keyPress) => handleKeyDown(keyPress)}
+      onClick={() => handleActive()}
+    >
       <ReactCollapsible
         trigger={closedTrigger}
         triggerWhenOpen={openTrigger}
         transitionTime={transitionTime}
-        tabIndex={0}
+        open={active}
       >
-        {children}
+        <div tabIndex={active ? 0 : -1}>{children}</div>
```

Comment:
`active` state is also used here to render the inner content tabbable when the Collapsible is open, and untabbable when the Collapsible is closed.

Reply:
Is there a way to make it so that the links are tabbable (which I assume was the driving force for this change) without making `<p>` tags tabbable? Maybe `tabIndex={active ? undefined : -1}` would do that? I honestly don't know

Reply:
Yeah, I originally made the `<p>` tags tabbable so that screen-readers are able to read the content. I'm actually not quite sure how people use screen readers, but I was using the ChromeVox extension for this. On the other hand, the links are tabbable regardless of making the content tabbable or not.

Reply:
Interesting! Mac's default screenreader VoiceOver doesn't have trouble reading them either way, but if ChromeVox is, maybe we could just hide the border that shows up on the focus state right now instead?

Reply:
If it works for Mac, it might work for other screen readers too? 🤷‍♀️ Does having the inner content be tabbable mess with the Mac Screen reader? If so, I can remove the focus altogether! Otherwise, just removing the focus outline works too!

Reply:
Ah nvm! You're right, it works in ChromeVox too. I'll remove the tabIndex from the content. 👍

Reply:
This should fix it - https://github.com/Dwolla/dev-portal/pull/106/commits/51f1819764827bc7b1a31862d3b378b6eb2f8f33